doc/source/ray-air/examples/feast_example.ipynb

```diff
@@ -1132,103 +1187,61 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "(GBDTTrainable pid=12634) 2022-06-02 14:26:23,018\tINFO main.py:980 -- [RayXGBoost] Created 1 new actors (1 total actors). Waiting until actors are ready for training.\n",
-      "(GBDTTrainable pid=12634) 2022-06-02 14:26:25,230\tINFO main.py:1025 -- [RayXGBoost] Starting XGBoost training.\n",
-      "(GBDTTrainable pid=12634) E0602 14:26:25.231635524   12691 fork_posix.cc:76]           Other threads are currently calling into gRPC, skipping fork() handlers\n",
-      "(_RemoteRayXGBoostActor pid=12769) [14:26:25] task [xgboost.ray]:139712002042896 got new rank 0\n"
-     ]
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Result for XGBoostTrainer_a3a2c_00000:\n",
-      "  date: 2022-06-02_14-26-26\n",
-      "  done: false\n",
-      "  experiment_id: 14b63d641b8e4583b3551b1af113ec6d\n",
-      "  hostname: ip-172-31-71-98\n",
-      "  iterations_since_restore: 1\n",
-      "  node_ip: 172.31.71.98\n",
-      "  pid: 12634\n",
-      "  should_checkpoint: true\n",
-      "  time_since_restore: 5.432286262512207\n",
-      "  time_this_iter_s: 5.432286262512207\n",
-      "  time_total_s: 5.432286262512207\n",
-      "  timestamp: 1654205186\n",
-      "  timesteps_since_restore: 0\n",
-      "  train-error: 0.09502400698384984\n",
-      "  train-logloss: 0.5147884634112437\n",
-      "  training_iteration: 1\n",
-      "  trial_id: a3a2c_00000\n",
-      "  validation-error: 0.1627094972067039\n",
-      "  validation-logloss: 0.5557328870197414\n",
-      "  warmup_time: 0.004194974899291992\n",
-      "  \n",
-      "Result for XGBoostTrainer_a3a2c_00000:\n",
-      "  date: 2022-06-02_14-26-31\n",
-      "  done: false\n",
-      "  experiment_id: 14b63d641b8e4583b3551b1af113ec6d\n",
-      "  hostname: ip-172-31-71-98\n",
-      "  iterations_since_restore: 81\n",
-      "  node_ip: 172.31.71.98\n",
-      "  pid: 12634\n",
-      "  should_checkpoint: true\n",
-      "  time_since_restore: 10.460175275802612\n",
-      "  time_this_iter_s: 0.03925156593322754\n",
-      "  time_total_s: 10.460175275802612\n",
-      "  timestamp: 1654205191\n",
-      "  timesteps_since_restore: 0\n",
-      "  train-error: 0.01802706241815801\n",
-      "  train-logloss: 0.07058723671627426\n",
-      "  training_iteration: 81\n",
-      "  trial_id: a3a2c_00000\n",
-      "  validation-error: 0.0824022346368715\n",
-      "  validation-logloss: 0.22556984905196217\n",
-      "  warmup_time: 0.004194974899291992\n",
-      "  \n"
+      "/home/ray/Desktop/workspace/ray/python/ray/util/placement_group.py:78: DeprecationWarning: placement_group parameter is deprecated. Use scheduling_strategy=PlacementGroupSchedulingStrategy(...) instead, see the usage at https://docs.ray.io/en/master/ray-core/package-ref.html#ray-remote.\n",
```

Comment:
hmm can we hide these?

Reply:
wondering why these are showing up...

Reply:
@jjyao seems we haven't finished this deprecation replacement.

Reply:
@richardliaw @ericl @jjyao These are fixed by #28315.